### PR TITLE
Add internationalization data and update Carbon components for language switch

### PR DESCRIPTION
**Pull request by meddhiaka**
# Pull Requests Requirements

- [X] The PR title includes a brief description of the work done, including the Issue number if applicable.
- [X] The PR includes a video showing the changes for the work done.
- [X] The PR title follows conventional commit label standards.
- [X] The changes confirm to the OpenElis Global x3 Styleguide and design documentation.
- [X] The changes include tests or are validated by existing tests.
- [X] I have read and agree to the Contributing Guidelines of this project.

## Summary

* I added internationalization data to thejson files in both French and English.
* I updated the carbon react components such as `input` and `select` to match the `useIntl` hook when switching languages

## Screenshots

image
image
image

## Related Issue
#926 

## Other

I will address another issue that has appeared while fixing this issue, which involves the presence of duplicate input elements within the same form like in this screenshot:

image

### CODE REVIEW

File: frontend/src/components/patient/CreatePatientForm.js [hunk 1]
```diff
@@ -383,12 +383,14 @@ function CreatePatientForm(props) {
                       value={values.subjectNumber || ""}
                       name={field.name}
                       labelText={intl.formatMessage({
-                        id: "patient.subject.number",
+                        id: "patient.first.name",
```
Comment (mozzy11) [inline frontend/src/components/patient/CreatePatientForm.js:386]: Why did you have to switch this code??

File: pom.xml [hunk 2]
```diff
@@ -577,6 +577,25 @@
 				<artifactId>maven-surefire-plugin</artifactId>
 				<version>2.18.1</version>
 			</plugin>
+			<plugin>
```
Comment (mozzy11) [inline pom.xml:580]: is this change part of this PR ??

Comment (meddhiaka) [reply pom.xml:580]: it is not, it's an issue in my development environment, i wanted to push it for another branch, ignore this commit

File: frontend/src/components/patient/CreatePatientForm.js (hunk 1 above)
Comment (meddhiaka) [reply frontend/src/components/patient/CreatePatientForm.js:386]: I will fix this thing

File: pom.xml (hunk 2 above)
Comment (meddhiaka) [reply pom.xml]: I reverted this commit, kindly resolve this conversation

File: frontend/src/components/patient/CreatePatientForm.js (hunk 1 above)
Comment (meddhiaka) [reply frontend/src/components/patient/CreatePatientForm.js]: I restored the old version:
https://github.com/I-TECH-UW/OpenELIS-Global-2/pull/930/commits/b26086aaf73f308d2d90374b1aa963bef5d601b2

File: frontend/src/components/patient/CreatePatientForm.js [hunk 3]
```diff
@@ -384,11 +384,13 @@ function CreatePatientForm(props) {
                       name={field.name}
                       labelText={intl.formatMessage({
                         id: "patient.subject.number",
+                        defaultMessage: "First Name",
```
Comment (mozzy11) [inline frontend/src/components/patient/CreatePatientForm.js:387]: First name is still a wrong default mesage . this is s field for Subject Number

File: frontend/src/components/patient/CreatePatientForm.js [hunk 4]
```diff
@@ -384,11 +384,13 @@ function CreatePatientForm(props) {
...
                       })}
                       id={field.name}
                       className="inputText"
                       invalid={errors.subjectNumber && touched.subjectNumber}
                       invalidText={errors.subjectNumber}
+                      placeholder={intl.formatMessage({id:"patient.information.firstname"})}
```
Comment (mozzy11) [inline frontend/src/components/patient/CreatePatientForm.js:393]: Looks like this is a wrong place holder for this field . its a subject NUmber

**Review by mozzy11 — APPROVED** [review]

**Pull request merged**